src/form.ts

```diff
@@ -15,7 +15,7 @@ export type FormValue = { [ key: string ]: any };
 function findFirstEnabledIndex(select: HTMLSelectElement) {
 	const length = select.options.length;
 	for (let i = 0; i < length; i++) {
-		if (!select.options[i].disabled) {
+		if (!(<HTMLOptionElement> select.options[i]).disabled) {
```

Comment:
I take it TS used to express `HTMLSelectElement`'s `options` as an array or collection of `HTMLOptionElement`s but 1.8 no longer does?  Seems like a let-down...